demo-app/src/ClipPathPreviewForm.css

```diff
@@ -0,0 +1,6 @@
+#shapeColorInput {
+    border-color: lightgray;
```

Comment:
Why is this not a Styled componet? We do not want external CSS file.

Reply:
Made the color input into a styled component.